subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cracen_mac_cmac.h

```diff
@@ -53,13 +53,3 @@ psa_status_t cracen_cmac_update(cracen_mac_operation_t *operation, const uint8_t
  * @return PSA_SUCCESS on success or a valid PSA status code.
  */
 psa_status_t cracen_cmac_finish(cracen_mac_operation_t *operation);
-
-/**
- * @brief Compute a CMAC in a single pass, without context switching.
- *
- * @note This function assumes the setup function is called first.
- *
- * @return PSA_SUCCESS on success or a valid PSA status code.
- */
-psa_status_t cracen_cmac_compute(cracen_mac_operation_t *operation, const uint8_t *input,
```

Comment:
Why is this removed? I see that the function is implemented in cracen_mac_cmac.c.

Reply:
Moved it over to SW since we only use it when using the workaround